core/src/main/java/org/kabeja/entities/Vertex.java

```diff
@@ -293,6 +296,23 @@ public void transform(TransformContext context) {
        
     }
 
+	/**
+	 * Transforms the arcs coordinates from OCS to WCS.
+	 * Does not simply translate the coordinates, but "reverts" the arbitrary
+	 * axis algorithm's effects.
+	 * Should only ever be called after the whole arc has been constructed
+	 * i.e. when the entity's block has ended.
+	 */
+    public void transformToWcs(Extrusion e) {
+		boolean hasDefaultExtrusion = e.getNormal().equals(new Vector(0,0,1));
+		//32: 3d polyline vertex; 64: 3d polygon mesh
+		boolean is3D = flags == 32 || flags == 64;
```

Comment:
I don't think this works like you think it should ;) but I don't really understand where these values are coming from anyways. Can you explain this next time you're here?

Reply:
From the dxf-doc:
"Vertex flags: [...] 32 = 3D polyline vertex, 64 = 3D polygon mesh"

I thought that these codes basically imply that the Vertex already uses WCS:
The doc also says that a vertex that is in 3D uses WCS, not OCS, which should make a transformation unnecessary.
I hope that makes sense :)

Reply:
discussion: use "flag-comparison" with &, maybe implement a method with a speaking name for this?